### PR TITLE
update server to use bibtex if a .bib file is present

### PR DESCRIPTION
**Pull request by mpacer**
This checks to see if a bib file is present and if it is it runs a command as is appropriate according to the `LatexConfig.bib_command` traitlet. 

eventually this should be factored out into a couple of synchronous functions for the setup too… but we don't need to worry about that for now.

### CODE REVIEW

File: jupyterlab_latex/__init__.py [hunk 1]
```diff
@@ -43,45 +44,74 @@ class LatexConfig(Configurable):
     """
     latex_command = Unicode('xelatex', config=True,
         help='The LaTeX command to use when compiling ".tex" files.')
+    bib_command = Unicode('bibtex', config=True,
+        help='The bibTeX command to use when compiling ".tex" files.')
```
Comment (ian-r-rose) [inline jupyterlab_latex/__init__.py]: Nitpicking here, but `BibTex` should be capitalized.

File: jupyterlab_latex/__init__.py [hunk 2]
```diff
@@ -43,45 +44,74 @@ class LatexConfig(Configurable):
...
+            command_sequence = [list(full_latex_sequence)]
+            
+            if any([re.match(r'.*\.bib', x) for x in set(glob.glob("*"))]):
+                command_sequence += [
+                    list(full_bibtex_sequence), 
+                    list(full_latex_sequence), 
+                    list(full_latex_sequence),
+                    ]
+                    
+            for i, cmd in enumerate(command_sequence):
```
Comment (ian-r-rose) [inline jupyterlab_latex/__init__.py]: It does not look like you are using `i` here, so we might as well do `for cmd in command`

Comment (mpacer) [reply jupyterlab_latex/__init__.py]: I was actually using the i inside the error but I can get around that.

Comment (ian-r-rose) [reply jupyterlab_latex/__init__.py]: Ah, my mistake! Consider this comment null and void, then.

**Pull request merged**